### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: app/pdl.yaml [hunk 1]
```diff
@@ -617,6 +617,386 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+  /pdl/titles/title-i/subtitles/subtitle-a/state-distribution:
```
Comment (sandeep-ps) [inline app/pdl.yaml:620]: Please update the version at the top of the YAML file.

Comment (ywkim312) [reply app/pdl.yaml:620]: updated.

**Comment by pengyin-shan** [comment]
Based on this PR, we will have `programPaymentInDollars`, `paymentInDollars`, `totalPaymentInDollars` among title I and title II (title II also has `subtitlePaymentInDollars`). Do they all mean *the total dollar amount for its corresponding level* ? If so, can we use `totalPyamentInDollars` instead of all different ones? it would be quite confusing if we use different attribute names for the same meaning in the long run

**Comment by ywkim312** [comment]
> Based on this PR, we will have `programPaymentInDollars`, `paymentInDollars`, `totalPaymentInDollars` among title I and title II (title II also has `subtitlePaymentInDollars`). Do they all mean _the total dollar amount for its corresponding level_ ? If so, can we use `totalPyamentInDollars` instead of all different ones? it would be quite confusing if we use different attribute names for the same meaning in the long run

Oh we can do it. If so, we will have to change other subtitles as well since the other subtitles are having the same structure. Also, if we want to change, we have start to from data-import again.

**Comment by pengyin-shan** [comment]
> > Based on this PR, we will have `programPaymentInDollars`, `paymentInDollars`, `totalPaymentInDollars` among title I and title II (title II also has `subtitlePaymentInDollars`). Do they all mean _the total dollar amount for its corresponding level_ ? If so, can we use `totalPyamentInDollars` instead of all different ones? it would be quite confusing if we use different attribute names for the same meaning in the long run
> 
> Oh we can do it. If so, we will have to change other subtitles as well since the other subtitles are having the same structure. Also, if we want to change, we have start to from data-import again.

I see, we can merge this PR and create the new release. For that release, let's talk about using the same attribute name for the values with the same meaning.

**Comment by ywkim312** [comment]
> > > Based on this PR, we will have `programPaymentInDollars`, `paymentInDollars`, `totalPaymentInDollars` among title I and title II (title II also has `subtitlePaymentInDollars`). Do they all mean _the total dollar amount for its corresponding level_ ? If so, can we use `totalPyamentInDollars` instead of all different ones? it would be quite confusing if we use different attribute names for the same meaning in the long run
> > 
> > 
> > Oh we can do it. If so, we will have to change other subtitles as well since the other subtitles are having the same structure. Also, if we want to change, we have start to from data-import again.
> 
> I see, we can merge this PR and create the new release. For that release, let's talk about using the same attribute name for the values with the same meaning.

Sounds good to me!

**Review by sandeep-ps — APPROVED** [review]
I briefly reviewed the code and ran the Postman collection. The PR looks good. It will be better to test with the deployed instance of the web app as well. I will approve this PR.

File: app/pdl.yaml (hunk 1 above)
Comment (sandeep-ps) [reply app/pdl.yaml:620]: Thanks.

**Review by pengyin-shan — APPROVED** [review]
This PR looks good to me. We can take a look at the API landscape after releasing this version and discuss possible improvements in the next release.

**Pull request merged**